api/capi/src/tensor_filter_single.c

```diff
@@ -352,7 +352,7 @@ g_tensor_filter_single_invoke (GTensorFilterSingle * self,
   if (G_UNLIKELY (!priv->fw) || G_UNLIKELY (!priv->fw->invoke_NN))
     return FALSE;
   if (G_UNLIKELY (!priv->fw->run_without_model) &&
-      G_UNLIKELY (!priv->prop.model_file))
+      G_UNLIKELY (!priv->prop.model_files || priv->prop.num_models < 1))
```

Comment:
1. ```||``` ==> ```&&``` and ```< 1``` ==> ```> 0```?
2. You are now not checking char*'s NULL. (i.e., model_files[0])

Reply:
Thanks, updated.